app/views/podcast_player/_form.html.erb

```diff
@@ -1,94 +1,149 @@
-<%= turbo_frame_tag "embed-player", data: {controller: "morphdom"} do %>
-  <div class="col-12 mb-4">
-    <div class="form-floating input-group" data-controller="dynamic-form">
-      <%= select_tag :embed_player_type, embed_player_type_options(player_options[:embed_player_type]), class: "form-control", data: {action: "dynamic-form#change"} %>
-      <%= label_tag :embed_player_type, t("helpers.label.podcast_player.embed_player_type") %>
-      <a hidden href="<%= request.fullpath %>" data-dynamic-form-target="link"></a>
+<div class="row my-4 mx-2" data-controller="embed-preview"
```

Comment:
I'd probably also split this out into partials.

Reply:
Updated to use episode player forms sub-partials.

Reply:
+1